### DISCUSSION TITLE
Blank Orders tab

### DISCUSSION THREAD

**Discussion by JohnWilkins82** [Erabior/RouteManager#104] [General]
Greetings, 
 Since the last Railroader update (2024.2.1) the Orders tab of the locomotive window is blank. removing the RouteManger mod fixes this issue.  Is there a way to get RouteManger to work or do I need to wait for a update? Thank you in advance and keep up the great work.

**Comment by AdamC1228** [comment]
This really should be opened as an issue rather than a discussion and we will need a copy of your log files.

**Comment by JohnWilkins82** [comment]
Understood. Was trying to determine if it was a widespread issue.  I found the LogOutput.log file in the beplinex folder. Are there any others I am missing?  I found the Player.log as well. 
Thank you for the reply.

**Reply by TnNytehawk** [reply]
I'm seeing the same issue as you, as well as a few other guys I talk to.
